credential/reclaim/v0.1.0/schema.json

```diff
@@ -0,0 +1,133 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/credential/reclaim/v0.1.0/schema.json",
+    "title": "Schema for Reclaim Protocol",
+    "description": "A schema for reclaim protocol credential",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#4bbf95",
+            "icon": "./icon.svg"
+        }
+    },
+    "allOf": [
+        {
+            "$ref": "https://core.schemas.verida.io/base/latest/schema.json"
```

Comment:
The Verida base schema is meant for records stored in the database which is not the case here.
This schema represents the credentialSubject property of a W3C Verifiable Credential.
The W3C Verifiable Credential will be wrapped in a record in the Credential database following the credential base schema.

Put simply, this schema should not extend the Verida base schema.

Reply:
You can have a look at the `identity/kyc/FinClusive/individual-basic` schema which is in a similar situation

Reply:
Got it. Removed